frame/sudo/src/lib.rs

```diff
@@ -122,8 +124,8 @@ decl_module! {
 		#[weight = SimpleDispatchInfo::FreeOperational]
 		fn sudo(origin, proposal: Box<T::Proposal>) {
 			// This is a public call, so we ensure that the origin is some signed account.
-			let sender = ensure_signed(origin)?;
-			ensure!(sender == Self::key(), "only the current sudo key can sudo");
+			let sender = ensure_signed(origin).map_err(|_| "origin must be signed")?;
```

Comment:
Maybe `ModuleDispatchError::as_str` should be used here? It gives a less readable error description, but at least the same string won't be used everywhere.